cmd/nexd/main.go

```diff
@@ -163,6 +187,55 @@ func main() {
 					},
 				},
 			},
+			{
+				Name:  "router",
+				Usage: "Enable child-prefix function of the node agent to enable prefix forwarding.",
+				Action: func(cCtx *cli.Context) error {
+					return nexdRun(cCtx, logger, nexdModeRouter)
+				},
+				Flags: []cli.Flag{
+					&cli.StringSliceFlag{
+						Name:     "child-prefix",
+						Usage:    "Request a `CIDR` range of addresses that will be advertised from this node (optional)",
+						EnvVars:  []string{"NEXD_REQUESTED_CHILD_PREFIX"},
+						Required: false,
+						Action: func(ctx *cli.Context, childPrefixes []string) error {
+							for _, prefix := range childPrefixes {
+								if err := nexodus.ValidateCIDR(prefix); err != nil {
+									return fmt.Errorf("Child prefix CIDRs passed in --child-prefix %s is not valid: %w", prefix, err)
+								}
+							}
+							return nil
+						},
+					},
+				},
+			},
+			{
+				Name:  "relay",
+				Usage: "Enable relay and discovery support function for the node agent.",
+				Action: func(cCtx *cli.Context) error {
+					if runtime.GOOS != Linux {
+						return fmt.Errorf("Relay node is only supported for Linux Operating System")
+					}
+
+					return nexdRun(cCtx, logger, nexdModeRelay)
+				},
+				Flags: []cli.Flag{
+					&cli.BoolFlag{
+						Name:     "enable-discovery",
```

Comment:
So the intention is to allow relay-only or relay+discovery? and not discovery-only?

It looks like docs need to be updated for this as well, right?